pkg/util/backoff_test.go

```diff
@@ -37,8 +37,12 @@ func TestSlowBackoff(t *testing.T) {
 	maxDuration := 50 * step
 
 	b := NewFakeBackOff(step, maxDuration, tc)
-	cases := []time.Duration{0, 1, 2, 4, 8, 16, 32, 50, 50, 50}
+	cases := []time.Duration{0, 1, 2, 4, 8, 16, 32, 50, 50, 50, 0}
 	for ix, c := range cases {
+		// The final test, we back down to zero: confirming that reset also works.
+		if ix == 10 {
```

Comment:
This seems fragile. I would rather see the cases remain as is, and then another set of tests for this after the for-loop. Or a separate test case completely.